README.md

```diff
@@ -111,12 +112,13 @@ Then continue with installing the native module
 
 ## Plugin Usage
 
-With the CodePush plugin downloaded and linked, and your app asking CodePush where to get the right JS bundle from, the only thing left is to add the necessary code to your app to control the following policies:
+After link the CodePush plugin, specify your app where CodePush can get the right JS bundle, you need to add necessary code in your app to tackle the below two issues: 
```

Comment:
This sentence is a little confusing, I would change "After link" to "After linking" and do a few more structural changes to this sentence.